lib/osf-components/addon/components/validation-errors/template.hbs

```diff
@@ -1,15 +1,20 @@
-{{#if this.isValidating}}
-    <LoadingIndicator @inline={{true}} @dark={{true}} />
-{{else}}
-    {{#if this.validatorErrors}}
-        <div
-            data-test-validation-errors={{@key}}
-            local-class='Errors'
-            ...attributes
-        >
-            {{#each this.validatorErrors as |error|}}
-                <p>{{~error~}}</p>
-            {{/each}}
-        </div>
+<div
+    {{did-insert this.setIsValidating @changeset.isValid}}
+    {{did-update this.setIsValidating @changeset.isValid}}
```

Comment:
This is to compensate for the lack of a `isValidating` property on the changeset object. The changeset object does have a `isValidating()` method but that means we cannot depend on it in a computed prop way. So a workaround (as was done before) is to watch `changeset.isValid` which is a getter that uses a tracked prop that gets updated upon as `changeset.validate()` [runs](https://github.com/validated-changeset/validated-changeset/blob/a23494a9a481fe639d4e00aaffdfc9e54ea50cce/src/index.ts#L249). `{{did-update}}` render modifier is rerun whenever its arguments change thus allowing us to recompute `changeset.isValidating()` and update `isValidating` tracked prop used here.

Reply:
That said, we don't need `@changeset.isValid` as argument for `{{did-insert}}` because it's only run once.
```suggestion
    {{did-insert this.setIsValidating}}
    {{did-update this.setIsValidating @changeset.isValid}}
```